src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(Color color, double radius) {
+        super(String.valueOf(color));

```

Comment:
Why did you keep a String type in base class, when you're using a Color instance everywhere else? Please fix this in base class.